events/events.go

```diff
@@ -0,0 +1,58 @@
+package events
```

Comment:
Maybe we should put this in the /libs directory ? (just my personal opinion on this point)